modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/css/ckeditor4/main.scss

```diff
@@ -335,3 +335,23 @@
 		padding: 0.5em;
 	}
 }
+
+.keyboard-message.popover {
```

Comment:
Is there a way to make this more specific to CKEditor, like some `.cke*` wrapper class? This class looks somewhat generic, so styles might spill over to some unrelated part of the page.